src/exampleTypes/Bool.js

```diff
@@ -15,4 +16,7 @@ let Bool = ({ tree, node, RadioList = RadioListDefault }) => (
   </div>
 )
 
-export default contexturify(Bool)
+export default _.flow(
+  contexturify,
+  withTheme
```

Comment:
we were using it without `withTheme` in two components: IMDBCards and FilterList's FieldLabel. IMDBCards was only used in quickStart (RIP ⚰️), so it can be safely deleted. 

Is there a reason for FieldLabel to exist on its own? It looks like it can just be merged with the Label component - I tried it and it seemed to work fine.

FieldLabel as of v2, for reference:
https://github.com/smartprocure/contexture-react/blob/d6fd56415d46d56e321e53fb79e6544a17811cac/src/FilterList.js#L158-L165

given that these are the only two exceptions, I'm gonna go ahead and roll it in. Should save us a bunch of code 🙂